Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -177,9 +183,12 @@ protected virtual void Dispose(bool disposing)
         private void InitializeForFrameworkEventSource()
```

Comment:
>InitializeForFrameworkEventSource [](start = 21, length = 33)

Now you have two implementations  - one that requires reflection and may fail to initialize and another that requires less. I do not see the fall back logic implemented. Am I missing something?